docs/tutorials/on-device-training.md

```diff
@@ -0,0 +1,1406 @@
+---
+title: On-Device Training
+nav_order: 11
+parent: Tutorials
+---
+
+# On-Device Training: Building an Android Application
+
```

Comment:
Add an image/preview of the android app at the top. This will make it easy for the users to see what they are about to build. You can use an existing image may be the last one with Tom Cruise.